addons/website_slides/static/tests/tours/slide_course_publisher_standard.js

```diff
@@ -99,6 +99,30 @@ wTourUtils.registerWebsitePreviewTour('course_publisher_standard', {
 }],
     slidesTourTools.addImageToSection('Introduction', 'Overview', true),
     slidesTourTools.addPdfToSection('Introduction', 'Exercise', true),
+    slidesTourTools.addQuizToSection(
+        'Introduction',
+        {
+            name: 'quiz',
+            questions: [
+                {
+                    title: 'In which year was this course created?',
+                    options: [{ title: '2024', isCorrect: true }, { title: '2023' }],
+                },
+                {
+                    title: 'Which one would you prefer more in your leisure time?',
+                    options: [{ title: 'Coding' }, { title: 'Sleeping', isCorrect: true }],
+                },
+            ],
+        },
+        true
+    ),
+    [
+        {
+            content: "eLearning: should not be able to input the title of a new question.",
```

Comment:
```suggestion
            content: "eLearning: wait for the page to no longer be in questions creation mode.",
```

Reply:
Hello @flch-odoo :wave: 
Done with the changes :+1: